source/develop/release-management/features/storage/DataCenterV4_1.html.md

```diff
@@ -0,0 +1,41 @@
+---
+title: DataCenterV4_1
+category: feature
+authors: mlipchuk
+wiki_category: Feature|DataCenterV4_1
+wiki_title: Features/DataCenterV4_1
+wiki_last_updated: 2016-09-20
+feature_name: Data Center V4.1
+feature_modules: engine/vdsm
+---
+
+## Summary
+#
+# Data Center 4.1 which was introduced at oVirt 4.1 should support the following features:
+#
+# * Storage Domain's version upgrade - Upgrading Data storage domains' version from V3 to V4
+# This should make the storage domains support qcow version 1.1 as well as the old qcow version 0.10.
+# * libgfapi
```

Comment:
Gluster libgfapi (perhaps add a link to http://staged-gluster-docs.readthedocs.io/en/release3.7.0beta1/Features/libgfapi/ ) ?